structuredevent-service-legacy/src/main/java/com/sequenceiq/cloudbreak/structuredevent/service/telemetry/mapper/ClusterUseCaseMapper.java

```diff
@@ -176,6 +178,10 @@ private UsageProto.CDPClusterStatus.Value lastStepToUseCaseMapping(String rootFl
                     useCase = getClusterStatus(nextFlowState, "CLUSTER_UPGRADE_PREPARATION_FINISHED_STATE",
                             UsageProto.CDPClusterStatus.Value.UPGRADE_PREPARE_FINISHED, UsageProto.CDPClusterStatus.Value.UPGRADE_PREPARE_FAILED);
                     break;
+                case "UpgradeRdsFlowEventChainFactory":
+                    useCase = getClusterStatus(nextFlowState, "UPGRADE_RDS_INSTALL_POSTGRES_PACKAGES_FINISHED_EVENT",
```

Comment:
The expected state should be "UPGRADE_RDS_FINISHED_STATE" state, not the event that triggers that state.